uint32 counter
time timestamp

float32[] global_xs
float32[] global_ys

float32[] local_xs
float32[] local_ys

float32[] local_trans_xs
float32[] local_trans_ys

float32 rx
float32 ry
float32 trans_rx
float32 trans_ry


